docs/configuration_file.md

```diff
@@ -270,6 +311,37 @@ Where:
 
 Pair converters are optional: if they are missing from the configuration file DaLI selects a default one.
 
+### CCXT
+This plugin is based on the CCXT Python library.
+
+Initialize this plugin section as follows:
+<pre>
+[dali.plugin.pair_converter.ccxt</em>]
+historical_price_type = <em>&lt;historical_price_type&gt;</em>
+fiat_priority = <em>&lt;fiat_priority&gt;</em>
+</pre>
+
+Where:
+* `<historical_price_type>` is one of `open`, `high`, `low`, `close`, `nearest`. When DaLi downloads historical market data, it captures a `bar` of data surrounding the timestamp of the transaction. Each bar has a starting timestamp, an ending timestamp, and OHLC prices. You can choose which price to select for price lookups. The open, high, low, and close prices are self-explanatory. The `nearest` price is either the open price or the close price of the bar depending on whether the transaction time is nearer the bar starting time or the bar ending time.
+* `fiat_priority` is an optional list of strings in JSON format that ranks the priority of fiat in the routing system. If no `fiat_priority` is given, the default priority is `["USD", "JPY", "KRW", "EUR", "GBP", "AUD"]`, which is based on the volume of the fiat market paired with BTC (ie. BTC/USD has the highest worldwide volume, than BTC/JPY, etc.).
```

Comment:
Can you mention the string format: is it comma-separated, space-separated, etc. Also update the default priority from JSON-list to the same format seen by the user.

Reply:
If we mention the default list here, we need to remember to update it, if we update it in the code: add a comment in the code reminding to update docs.

Reply:
than -> then

Reply:
I gave an example of JSON format which hopefully makes it clear.

Reply:
I also added a comment to the CCXT pair converter PR #53

Reply:
The string in JSON format is what was confusing me, because it's in the context of a .ini file. I was thinking we could use a comma-separated list of values without white space, but it's not terribly pretty: let's go with your JSON line.